### ISSUE TITLE
Generate method does not create new block for record

### ISSUE THREAD

**Issue by jmarolf** [dotnet/roslyn#48326]
Given the code below:
```csharp
using System;
public class Program
{
    public static void Main()
    {
        var person = new Person("Jon");
        person.SayHello();
    }
}

record Person(string Name);
```
invoke the codefix to generate a method for `SayHello`

**Expected**
```csharp
using System;
public class Program
{
    public static void Main()
    {
        var person = new Person("Andrey");
        person.SayHello();
    }
}

record Person(string Name)
{
    internal void SayHello()
    {
        throw new NotImplementedException();
    }
}
```

**Actual**
```csharp
using System;
public class Program
{
    public static void Main()
    {
        var person = new Person("Andrey");
        person.SayHello();
    }
}

record Person(string Name) internal void SayHello()
{
    throw new NotImplementedException();
};
```

**Comment by jmarolf** [comment]
CC: @CyrusNajmabadi

**Comment by Youssef1313** [comment]
Fixed in #48313

**Comment by CyrusNajmabadi** [comment]
Closing as resolved.
